css/style.css

```diff
@@ -9,9 +9,11 @@ table {
 .ui-draggable, .ui-droppable {
 	background-position: top;
 }
-
+a {
+	color:blue !important;
+}
 p.small {
-   line-height: 
+	line-height:
```

Comment:
We should either fix or remove this incomplete property specification. (I know that wasn't part of the original intent of this PR).

Reply:
I added a .9em to complete the property.